operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,236 @@
+    merge-mode: "squash-merge"
+    my_pipeline1:
+      jobs:
+        - my_job1
+        - my_job2
+        ......
+    my_pipeline2:
+      jobs:
+        - my_jobs
+    ...
+
+```
+
+By default zuul will observe all branches for such files. We have to set the repository name
+that have to match the exact value that was set for zuul. Set a default-branch where actions
+that don't match an explicit branch are executed on. Set the merge-mode that zuul has to use.
+But beware that not all issue tracker support all methods. For github squash-merge will work.
+
+After these three properties add the pipelines you want to use to the project definition.
+With the `jobs` list you define which jobs to run in which pipeline.
+
+### Pipelines
+
+Every zuul instance will have at least one repository that is used for configuration. There
+you will find the available pipelines. Pipelines are used to run your jobs on a periodic or
+event driven base. Pipelines can be used to run other pipelines and to keep your jobs in a
+defined order if you need this.
+
+Have a look at the configuration repository to utilize the pipelines for your repository. You
+are not able to define new pipelines outside of a so called "configuration" repository. Since,
+by default your repo is considered "untrusted". So in the first place you don't need to
+think about, how to create a pipeline. Just use one that fits your needs as close as possible.
+
+### Jobs
+
+All jobs that your zuul instances knows of can be used for your own purposes.
+Call them directly or implement a job that uses an existing job as parent.
+Didn't find the right job? Than we have to create a new one. Existing jobs
+can be found in the web ui of your zuul instance: [Example](https://zuul.scs.community/t/SCS/jobs)
+
+First have a look on a basic job example:
+
+```yaml
+- job:
+    name: base
+    parent: null
+    description: |
+      The recommended base job.
+
+      All jobs ultimately inherit from this.  It runs a pre-playbook
+      which copies all of the job's prepared git repos on to all of
+      the nodes in the nodeset.
+
+      It also sets a default timeout value (which may be overidden).
+    pre-run: playbooks/base/pre.yaml
+    post-run:
+      - playbooks/base/post.yaml
+      - playbooks/base/post-logs.yaml
+    roles:
+      - zuul: zuul/zuul-jobs
+    timeout: 1800
+    nodeset:
+      nodes:
+        - name: ubuntu-jammy
+          label: ubuntu-jammy
+
+```
+
+Each jobs needs a name. Each job need to define whether there is parent job or not.
+Jobs without a parent are called "base" jobs. Usually you don't want to implement base jobs since
+there are already some base jobs that implement often used stuff. A description may not be mandatory
+but is obviously useful.
+
+Necessary for zuul to do anything you just need to add a `run` or `roles` property. Within something that is
+like a `noop` job or just printing something to stdout that is everything you need to run your first job.
```

Comment:
I don't understand the sentence beginning with "Within something", and it would be nice to see an example for that as well.

Reply:
something should mean job 
i updated the text